mac/build.sh

```diff
@@ -12,80 +12,38 @@ KEYMAN_MAC_BASE_PATH="$KEYMAN_ROOT/mac"
 . "$KEYMAN_ROOT/resources/build/build-help.inc.sh"
 . "$KEYMAN_ROOT/mac/mac-utils.inc.sh"
 
-# This script runs from its own folder
-cd "$(dirname "$THIS_SCRIPT")"
-
+builder_describe "Builds Keyman for macOS." \
+  "@/core:mac" \
+  "clean" \
+  "configure" \
+  "build" \
+  "publish   Publishes debug info to Sentry and builds DMG, .download_info for distribution" \
+  "test" \
+  "install   Installs result of Keyman4MacIM locally." \
+  ":engine   KeymanEngine4Mac" \
+  ":im   Keyman4MacIM" \
```

Comment:
Shall we go with `:app` as it matches other platforms better? The distinction is a little murky, I know.

Reply:
I second `:app` for consistency across the platforms